README.md

```diff
@@ -106,6 +107,65 @@ $ crossplane beta render xr.yaml composition.yaml functions.yaml
 See the [composition functions documentation][docs-functions] to learn more
 about `crossplane beta render`.
 
+### ExtraResources
+
+By defining one or more special `ExtraResources`, you can ask Crossplane to
+retrieve additional resources from the local cluster and make them available to
+your templates. See the [docs][extra-resources] for more information.
+
+```yaml
+apiVersion: meta.gotemplating.fn.crossplane.io/v1alpha1
+kind: ExtraResources
+requirements:
+  some-foo-by-name:
+    # Resources can be requested either by name
+    apiVersion: example.com/v1beta1
+    kind: Foo
+    matchName: "some-extra-foo"
```

Comment:
Hey There, I just tried to match a Namespaced Resource by name but it appears this isnt possible for namespaced resources. Is this assumption right? 

This is the corresponding error: 

projectname is a variable i'm trying to assign the result to

```
        cannot compose resources: fetching resources for projectName: cannot get
        extra resource by name: an empty namespace may not be set when a
        resource name is provided
```

this is my extra resource:

```
apiVersion: 'meta.gotemplating.fn.crossplane.io/v1alpha1'
kind: 'ExtraResources'
requirements:
  projectName:
    apiVersion: example.com/v1alpha1
    kind: ExampleProject
    matchName: {{ $spec.projectName }}
```

Reply:
hi, yes, only cluster scoped resources are supported